taichi/runtime/runtime.cpp

```diff
@@ -1143,11 +1144,11 @@ struct printf_helper {
 };
 
 template <typename... Args>
-void Printf(const char *format, Args &&... args) {
+void taichi_printf(Runtime *runtime, const char *format, Args &&... args) {
 #if ARCH_cuda
   printf_helper helper;
   helper.push_back(std::forward<Args>(args)...);
-  vprintf((Ptr)format, helper.ptr());
+  runtime->vvprintf(format, (const char *)helper.ptr());
 #else
   printf(format, args...);
```

Comment:
Here we should use `runtime->vvprintf`

Reply:
Use of `vvprintf` caused a Travis CI fail.